src/content/docs/release-notes/mobile-release-notes/net-maui-release-notes/net-maui-agent-004.mdx

```diff
@@ -0,0 +1,9 @@
+---
+subject: .NET MAUI agent
+releaseDate: '2024-01-04'
+version: 0.0.4
+downloadLink: 'https://www.nuget.org/packages/NewRelic.MAUI.Plugin'
+---
+
+# New in this release
+* Fixed issues related iOS Dsyms Script
```

Comment:
```suggestion
## New in this release

* Fixed issues related iOS dSYM Script
```

Reply:
@ndesai-newrelic Can you clarify what this bullet point means? My guess is you mean "fixed issues related to a run script for uploading DSYM files." 

Is that correct?